shortcircuit-api/src/main/java/com/powsybl/shortcircuit/json/ShortCircuitParametersSerializer.java

```diff
@@ -27,9 +28,9 @@ public ShortCircuitParametersSerializer() {
     public void serialize(ShortCircuitParameters parameters, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
         jsonGenerator.writeStartObject();
         jsonGenerator.writeStringField("version", ShortCircuitParameters.VERSION);
-        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withLimitViolations", parameters.isWithLimitViolations(), false);
-        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withVoltageMap", parameters.isWithVoltageMap(), false);
-        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withFeederResult", parameters.isWithFeederResult(), false);
+        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withLimitViolations", parameters.isWithLimitViolations(), DEFAULT_WITH_LIMIT_VIOLATIONS);
+        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withVoltageMap", parameters.isWithVoltageMap(), DEFAULT_WITH_VOLTAGE_MAP);
+        JsonUtil.writeOptionalBooleanField(jsonGenerator, "withFeederResult", parameters.isWithFeederResult(), DEFAULT_WITH_FEEDER_RESULT);
```

Comment:
Not sure this is good practice. In JSON (serialization in general), we can consider that no field for a boolean means it is false. However I agree there is a bug but this should not be the fix. The cleanest way would be for me to **always** serialize these fields. The second option is to fix the deserializer so that no field means `false`.

Reply:
It's ok for me to always serialize these fields. I made the change